.circleci/config.yml

```diff
@@ -534,6 +534,39 @@ jobs:
       - store_test_results:
           path: build/test-results
 
+  verify-revenuecatui-snapshot-tests:
+    description: "Verify RevenueCatUI snapshot tests"
+    <<: *android-executor
+    steps:
+      - checkout
+      - install-sdkman
+      - revenuecat/install-gem-unix-dependencies:
+          cache-version: v1
+      - android/restore-build-cache
+      - run:
+          name: Build and run RevenueCat UI snapshot tests
+          command: |
+            bundle exec fastlane android verify_revenuecatui_snapshot_tests
+      - android/save-build-cache
+
+  record-revenuecatui-snapshot-tests:
```

Comment:
And this isn't really recording the tests, but the snapshots, so maybe `record-revenuecatui-snapshots`?